src/vm/ClrEtwAll.man

```diff
@@ -1761,6 +1775,29 @@
                         </UserData>
                     </template>
 
+                    <template tid="ResolutionAttempted">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="AssemblyName" inType="win:UnicodeString" />
+                        <data name="Stage" map="ResolutionAttemptedStageMap" inType="win:UInt16" />
+                        <data name="AssemblyLoadContext" inType="win:UnicodeString" />
+                        <data name="Result" map="ResolutionAttemptedResultMap" inType="win:UInt16" />
```

Comment:
I'm wondering if it would make sense to change `Result` to `Status`. That could leave the door open to re-using this event to log both the start and end of a stage at a later point (which could alleviate the concerns @vitek-karas raised about expectations around getting an event once some operation has started, especially in the live scenario).

Thoughts?

Reply:
I'm starting to think that we don't need to make the traces absolutely super easy to read - it's an advanced scenario after all. So I think it would be fine if I could deduce all the "start" information from the previous event (whatever that is), then there's no need to add the explicit "start" event for each stage/extension point. I think it's OK to expect general knowledge of the algorithm and thus sequence of the events. At least for now.
If the "Start" event doesn't actually add any new information, then for now I would leave it out - I don't want to overdo the tracing (it's always easier to add than remove).

Reply:
I would agree with leaving the 'start' of the stage out for now. I think part of my question was more around naming - if it would make sense to name the property something that could - if we find a need to add it in the future - represent the start (as well as the result as it currently does)

Reply:
I added a Start event because it should be possible to use it to time each step.  (No worries in removing it, though.)